<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordSetting</class>
 <widget class="QDialog" name="PasswordSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgb(255, 255, 255);
}
QLineEdit {
	color:#ADADAD;
    border: 0px solid rgb(41, 57, 85);
    border-radius: 4px; 
    background: #EDEEF0;  
    selection-background-color: green; 
	font-family: &quot;Microsoft YaHei&quot;;
    font-size: 16px ;
	padding-left:20px;
	padding-right:20px;
}
 
 QLineEdit[echoMode=&quot;2&quot;] {
      lineedit-password-character: 9679;
}

QLineEdit:hover {
    border: 1px solid rgba(71,107,253,1);; 
}

QPushButton#pushButton_next{
	background-color:rgb(71,107,253);
	border-width:0;
    padding:4px;  
	border-style:inset;  
	border-radius:10px;  
	color:white;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 20px;
}

QPushButton#pushButton_next:pressed {  
    /* 改变背景色   
    background-color:rgb(40,85,20); */
    /* 使文字有一点移动 */  
    padding-left:6px;  
    padding-top:6px;  
}  


</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font-family: &quot;Microsoft YaHei&quot;;
	font:bold 24px;
    color: #333333;</string>
   </property>
   <property name="text">
    <string>Password Setting</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>312</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_confirm">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>312</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_next">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>312</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>270</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font-family: &quot;Microsoft YaHei&quot;;
	font-size:10px;
    color: #999999;</string>
   </property>
   <property name="text">
    <string>12 Digits Limited,Only English And Number Supported.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
